### PR TITLE
Fix spellings and a minor typo

### PR DESCRIPTION
**Pull request by fhdkhan**

### CODE REVIEW

**Comment by jr0cket** [comment]
Thank you for raising this pr, however its too big for me to merge as one pr.

I'll review the suggestions and make updates to the book

File: clojure-editors/editor-guides/setup.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ If anyone attending does not have access to a laptop running one of the above ch
```
Comment (jr0cket) [inline clojure-editors/editor-guides/setup.md:11]: This is a redundant file that should be deleted instead.

File: clojure-editors/editor-user-guides/vscode-calva.md [hunk 2]
```diff
@@ -47,10 +47,10 @@ Once you have a running REPL, use these commands to help you develop your code.
 
 | Evaluate code             | Keybinding         | Description                                                          |
 |---------------------------|--------------------|----------------------------------------------------------------------|
-| Namespace/file            | `Ctrl+Alt+c ENTER` | Loead the current file/namespace and any namespaces it depends on    |
```
Comment (jr0cket) [inline clojure-editors/editor-user-guides/vscode-calva.md:50]: This breaks the table formatting unfortunately.

File: clojure-editors/editor-guides/setup.md (hunk 1 above)
Comment (fhdkhan) [reply clojure-editors/editor-guides/setup.md:11]: Yes I noticed that but I was not sure which one you would like to keep so I didn't touch it.

Comment (fhdkhan) [reply clojure-editors/editor-guides/setup.md:11]: Also please feel free to squash the commits in one commit when you decide to merge.

File: clojure-editors/editor-user-guides/vscode-calva.md (hunk 2 above)
Comment (fhdkhan) [reply clojure-editors/editor-user-guides/vscode-calva.md:50]: Can you please elaborate how this breaks the table formatting. I think I just removed a character. Is it introducing something unintentional?

**Comment by fhdkhan** [comment]
Close to break it in small pieces which are easily reviewable

**Pull request closed without merging**